source/Interfaces/Menu.cpp

```diff
@@ -0,0 +1,126 @@
+/**
+ * @file Menu.cpp
+ * @author
+ */
+
+#include "Menu.hpp"
+#include <SFML/Window.hpp>
+
+namespace i_2D{
+    Menu::Menu(float width, float height){
+
+        //menu[0].setFont(font);
+        menu[0].setFillColor(sf::Color::Red);
+        menu[0].setString("Play");
+        menu[0].setPosition(sf::Vector2f(width / 2, height / MAX_NUMBER_OF_ITEMS_MENU_ITEMS +1) * 1.f);
+
+        //menu[1].setFont(font);
+        menu[1].setFillColor(sf::Color::White);
+        menu[1].setString("Options");
+        menu[1].setPosition(sf::Vector2f(width / 2, height / MAX_NUMBER_OF_ITEMS_MENU_ITEMS +1) * 1.f);
+
+        //menu[2].setFont(font);
+        menu[2].setFillColor(sf::Color::White);
+        menu[2].setString("Exit");
+        menu[2].setPosition(sf::Vector2f(width / 2, height / MAX_NUMBER_OF_ITEMS_MENU_ITEMS +1) * 1.f);
+
+        selectedItemIndex = 0;
+    }
+
+    Menu::~Menu() {
+    }
+
+    void Menu::draw(sf::RenderWindow &window) {
```

Comment:
This function might need to be called `draw` due to something in SFML, but if not then you should call it `Draw` (capital 'D') for consistency.